ctypes_override/__init__.py

```diff
@@ -0,0 +1,552 @@
+"""create and manipulate C data types in Python"""
```

Comment:
Now that you're no longer patching, it would be useful to see a blurb about why this is necessary and what was changed in this override here.

Reply:
where would the blurb live? `snap/snapcraft.yaml` or inside this same file; I might go to patching again since I figured I was doing it wrong the other day

Reply:
Yeah I would say inside this same file. But going back to patching is also a good option.